src/exercism/ResistorColorDuo.java

```diff
@@ -0,0 +1,34 @@
+class ResistorColorDuo {
+    int value(String[] colors) {
+        if (colors.length < 2) {
+            throw new IllegalArgumentException("At least two colors");
+        }
+
+        int firstDigit = -1;
+        int secondDigit = -1;
+
+        String[] colorList = {"black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"};
```

Comment:
There is a rule in programming that if a variable doesn't change, it is a constant, and we use UPPER_CASE format for it

Reply:
what about this comment?